network/ServerNetworking.cpp

```diff
@@ -30,8 +30,35 @@ namespace {
 class DiscoveryServer {
 public:
     DiscoveryServer(boost::asio::io_context& io_context) :
-        m_socket(io_context, udp::endpoint(udp::v6(), Networking::DiscoveryPort()))
-    { Listen(); }
+        m_service(io_context),
```

Comment:
`m_service` is only used in the constructor, so does it need to be stored as a member reference?

Reply:
Fixed.